app/assets/stylesheets/navbar.css

```diff
@@ -0,0 +1,25 @@
+.navbar {
+    background-color: #d5ad7eff;
```

Comment:
If any of these colors will be reused, consider creating color variables! This also helps with readability rather than trying to decipher hex codes

To do this, I recommend creating your CSS files with the `.scss` extension for extra functionality
Define a color variable as
```
$dark-blue: #d5ad7eff;
```
Then anywhere you want to use that color
```
background-color: $dark-blue;
```